java/ddlogapi.c

```diff
@@ -192,6 +192,96 @@ JNIEXPORT jlong JNICALL Java_ddlogapi_DDlogAPI_ddlog_1run(
     return handle;
 }
 
+ddlog_log_destination log_config(JNIEnv *env, jint mode, jstring addr_str) {
+    switch (mode) {
+        // socket
```

Comment:
idk anything about the context here, but in an ideal world these would be some kidn of enummeration {socket, disk, ...}

Reply:
Yah, this does look like something that should be handled by `cbindgen`, but that's probably out of scope for this PR

Reply:
This is a Java-to-C bridge, not C-to-Rust, so `cbindgen` wouldn't help.

Reply:
@convolvatron , good point,  I can't use Java enum's from C (without writing even more ugly JNI code), but I have C enums I could use here, and in the other location you pointed out.